### PR TITLE
Rs 2022 03 18

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 1]
```diff
@@ -6,32 +6,42 @@
...
-It's may be hard to feel at ease around her, but she's still a professional after all.
+I may have a bit of a hard timing feeling at ease around her,
+ but I can't deny that she's a professional.
 }
 [sha:ed374f959f62039afa4d97ffc97d83b666af4700]{
 -- Page 49, Offset 21438.
 -- 外科専門なので外傷も診てもらえる。
-Since she specializes in surgery, I should have her check on my wound as well.
+// 外傷 is explicitly _external_ wounds but then in the next sentence he's on to _internal_ bleeding, feels a bit non-sequiturial
+// Also is the 外傷 in question supposed to be his chest?
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:22]: I think so. I wonder if "wound" is a little nonspecific? Maybe we should specify?

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 2]
```diff
@@ -6,32 +6,42 @@
...
 -- Page 50, Offset 21440.
 -- 「ご指名と聞いてやってきたけどぉ。
-"I came here because I heard you called for me.
+"I came because I heard you'd requested me personally, but...
 }
 [sha:1ea5f576a8d6a61b1d5ee0042bfd70475f9c6203]{
 -- Page 50, Offset 21441.
 -- お客さん、悪いところは見当たらないわよぅ？　腕の一本、目玉の一つでもイッちゃってるかと期待して飛んできたのに、これじゃアタシ欲求不満よぅ？　あ、性的な意味もあるけど、おもに知識欲の話ねぇ？」
-But customer, I don't see anything wrong here. I rushed over here expecting an arm or an eye to be blown out, but seeing this makes me rather frustrated. Ah, you could take that as being sexual, but I was mainly referring to my intellectual thirst."
+// Reading お客さん more as a joking honorific than as literal?
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:39]: definitely

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 3]
```diff
@@ -56,52 +68,70 @@ The results were all normal. No problems.
 [sha:00eade398fb36e0172dac454a9f0b0c1253c1e7a]{
 -- Page 52, Offset 21446.
 -- 阿良句博士が“アタシったら無駄骨チャーン！”と唇をとがらせるのもやむなしだ。
-Professor Arach couldn't help but pout as she shouts, 'What am I, Worthless-chan?!'
+// Sisyphean is more like an _endless_ task than a _futile_ one, but it's not far off?
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:71]: Well, I appreciate the time you put into this. but your commenting self is right, sisyphean isn't quite right. Of the options below, some of which are cute and some of which hurt me, I _guess_ I prefer "frau fruitless"... However, I wonder if we shouldn't just go back to something like "waste-of-time-chan" just cop out.

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 4]
```diff
@@ -56,52 +68,70 @@ The results were all normal. No problems.
...
+...I guess she's talking about the extract derived from a woman's placenta.
 }
 [sha:98fe7ecb4ea3077fffdbdb53445cf7e486e90e2c]{
 -- Page 54, Offset 21451.
 -- 肌をよくする美容効果が主とされるが、実際は疲労回復の促進剤でもあり、老化を抑える効果もあるという。
-It's mainly used for its cosmetic effect of improving the skin, but in reality, it can be used as a stimulant for exhaustion recovery and it also has the effect of reducing aging.
+// TL note: nothing in this VN is medical advice
+People mostly use it as a beauty treatment to improve their skin,
+ but it can actually also help you recover from fatigue,
+ and even combat the effects of ageing.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:107]: >  Aging is the only appropriate spelling for American English and Canadian English. Ageing is the standard spelling for British English

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 5]
```diff
@@ -201,27 +244,35 @@ Coupled with her exposed garments, it honestly puts me at odds with my sexual de
...
-"Oh yeah that's right. My bad, my bad.
+"Oh, so I was. My bad, my bad.
 }
 [sha:88b7075b3b32febcfa325d577e5a7a6679735120]{
 -- Page 65, Offset 21476.
 -- ……ふーん。ほーう。なるほどぅ……なんか使ったにせよ、完全に塞がってるわー。貧弱なのに代謝機能は人並みというか、生き残りやすいというかぁ。あーあー、すっっっごく残念。志貴チャン、何も引くものがないんだもの。これじゃあアタシも足しようがないって感じぃ？」
-...Hmm. Hoh. I see... Whatever they used, they closed it off completely. Even though the quality is poor, perhaps your metabolic function is average, or maybe you can survive easily. Aaah, how utterly disappointing. There's nothing to take away from Shiki-chan. With this, it's like there's nothing more that I can add, right?"
+...Hmm. Hoh. I see...
+ I'm not sure that they used, but it seems to have healed over nicely.
+ You may be frail, but perhaps your metabolic functions are average after all. Or maybe it was just that survivable.
```
Comment (Avuxo) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:254]: metabolic functions are ordinary after all.

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 9/09_00_ARC09_2_1.txt:22]: Looking back on it I'm less sure, I feel like he uses 傷 instead when he's talking about that. Might just leave it as `examine my wounds`.

Comment (yk1512) [reply script/Arcueid/Day 9/09_00_ARC09_2_1.txt:22]: It sounds to me as if he's making a distinction between the two types of injuries, his 'wounds' being surface level injuries that he definitely has and the 'internal bleeding' being the internal level that he might not have but wants checked anyway.

**Review by yk1512 — APPROVED** [review]

**Pull request merged**